README.md

```diff
@@ -82,12 +82,12 @@ safety, but are not sufficient by themselves.
 
 **SES**, or *Secure EcmaScript*, is a dynamic, absorbed subset of
 ES-strict. To achieve this subsetting, SES builds on [Frozen
-Realms](https://github.com/tc39/proposal-frozen-realms/) which builds
-on [Realms](https://github.com/tc39/proposal-realms/). (Shims at
+Realms](https://github.com/tc39/proposal-ses) which builds
+on [[Shadow]Realms](https://github.com/tc39/proposal-shadowrealm). (Shims at
 [Realms
 shim](https://github.com/Agoric/realms-shim) and
 [Frozen Realms
-shim](https://github.com/tc39/proposal-frozen-realms/tree/master/shim).)
+shim](https://github.com/endojs/endo/tree/master/packages/ses).)
```

Comment:
This probably needs fresh words since SES now uses in-realm `lockdown()` with Compartments instead of Frozen Realms, but I'm not entirely sure what exactly to write.

Reply:
#106